ls1intum/Artemis
# `Assessment`: Make example submissions available for editors

### Checklist
#### General
#### Server

### Motivation and Context
When Editors try to create an example submission used for assessment training, a 404 is thrown and the editor doesn't have access to the assessed example submission.

### Description
On the server side, there is currently only a distinction between Instructor and Tutor. While the Instructor always gets the Example Assessment, a 404 is thrown for the Tutor and Assessment Training. The Tutor only gets the Example Assessment, if it is not used for Assessment Training.
This PR only makes a distinction between editor and tutor. Editors (and subsequently Instructors) always get the example submission, while tutors only get the example submission, if it is not used for assessment training.

Also updated the documentation: https://artemis-platform--5566.org.readthedocs.build/en/5566/admin/accessRights/

### Steps for Testing

Prerequisites:
- 1 Editor

1. Log in to Artemis
2. Navigate to Course Administration -> Exercise
3. Create a new Modelling Exercise
4. Create a Example Submission used for assessment training
5. Store the example submission, navigate one page back and again into the example submission
6. Verify, that no Error (404) is thrown and that the example assessment is visible

### Review Progress





#### Code Review
#### Manual Tests
